content/working_with/console/widgets/deploymentsView.md

```diff
@@ -92,14 +92,15 @@ deployment.
 
 In the header of the right side are:
 
-- the currently selected deployment name
+- the currently selected deployment
+- button to drill-down to development details
 - buttons to drill-down to child environments or services
 
-    The buttons will be disabled when there are no child environments or
```

Comment:
We probably need to specify which buttons are not visible (as Details button will always be visible) 🙂 

Maybe you can merge this part with the last bullet somehow? Having e.g.: 
```
- buttons to drill-down to child environments or services (only visible when there are child environments or services)
```

Reply:
I thought about it, and inside the `deploymentsView.md` file there are additional indents, which makes the information: 
```
The buttons will be disabled when there are no child environments or...
```

Appear underneath the `buttons to drill-down to child environments or services` bullet    
(as shown on the screenshot)
![Bullet point](https://user-images.githubusercontent.com/89005877/142199440-bc18c48d-e92f-470b-8526-3b542338cd6c.png)

What do you think about renaming the sentence from:
```
The buttons will not be visible when there are no child environments or services.
```

To:
```
Those buttons will not be visible when there are not child environments or services.
```

? :)

I think that would strenghten the correlation, that we are getting with the usage of additional indentation underneath the bulletpoint, making the context of buttons more specific 😎

Reply:
Oh, if "The buttons will not ..." is indented, then I think we can keep it or change it according to your suggestion ("Those buttons..."). But right now I see it is not indented 🙂, so that's where my confusion comes from: 
![image](https://user-images.githubusercontent.com/5202105/142211774-88aac209-5c16-4795-9e17-cd5c910ee5ef.png)

BTW: Documentation from branches is available at https://docs.cloudify.co/staging/<BRANCH_NAME> , so in your case I looked at: https://docs.cloudify.co/staging/RD-3216/working_with/console/widgets/deploymentsview/#drilling-down-to-child-deployments

Reply:
As multiple comments were addressing the issue related to the whitespaces overwriting (by the helpful IDE 😁)
I've restored old whitespace's, and double checked pointed out places in the documentation with the usage of hugo

Additionally, thanks for the tip related to the building the documentation site / viewing changes that may appear there! :)

Reply:
Thanks. Looks good 👍🏻